LCM_of_array_of_elements/LCM.c

```diff
@@ -0,0 +1,45 @@
+// C program to find LCM of n elements
```

Comment:
Fix file name and folder name to `largest_common_multiple`.

Reply:
Please solve this @ritish099